pkg/backup/backup/backup_manager.go

```diff
@@ -387,6 +388,18 @@ func (bm *backupManager) makeBackupJob(backup *v1alpha1.Backup) (*batchv1.Job, s
 		},
 	})
 
+	// mount volumes if specified
+	if backup.Spec.Local != nil {
+		localVolume := backup.Spec.Local.Volume
+		localVolumeMount := backup.Spec.Local.VolumeMount
+		if localVolumeMount.Name != localVolume.Name {
```

Comment:
Validate this in https://github.com/pingcap/tidb-operator/blob/master/pkg/backup/util/util.go#L352.

Reply:
no need to validate local storage provider, because volume/volumeMount will be validated by Pod.

Reply:
moved validation logic to `validateBackup()`